src/services/ApiService.ts

```diff
@@ -222,6 +223,68 @@ const getContractCallQuote = async (
   }
 }
 
+const getContractCallsQuote = async (
+  requestConfig: ContractCallsQuoteRequest,
+  options?: RequestOptions
+): Promise<LifiStep> => {
+  const config = ConfigService.getInstance().getConfig()
+
+  // validation
+  const requiredParameters: Array<keyof ContractCallsQuoteRequest> = [
+    'fromChain',
+    'fromToken',
+    'fromAddress',
+    'toChain',
+    'toToken',
+    'toAmount',
+    'contractCalls',
+  ]
+
+  if (requestConfig.contractCalls.length === 0) {
+    throw new ValidationError(`Parameter "contractCalls" is empty.`)
+  }
+
+  requiredParameters.forEach((requiredParameter) => {
+    if (!requestConfig[requiredParameter]) {
```

Comment:
wdyt about using "in" operator instead of the if check ? 

`if (requiredParameter in requestConfig)`